bindings/python/py_src/tokenizers/implementations/sentencepiece_unigram.py

```diff
@@ -0,0 +1,42 @@
+from tokenizers import Tokenizer, AddedToken, pre_tokenizers, decoders, trainers
+from tokenizers.models import Unigram
+from tokenizers.normalizers import NFKC
+from .base_tokenizer import BaseTokenizer
+
+from typing import Optional, List, Union
+
+
+class SentencePieceUnigramTokenizer(BaseTokenizer):
+    """SentencePiece Unigram Tokenizer
+
+    Represents the Unigram algorithm, with the pretokenization used by SentencePiece
+    """
+
+    def __init__(
+        self, vocab: Optional[str] = None, replacement: str = "▁", add_prefix_space: bool = True,
```

Comment:
Are you sure ? 

is_spm_files corresponds to a file given after `spm_export_vocab` command. It's doable but I think it will confuse users more than help, as most users will point to `.model` files if `is_spm_file` is set to `True`. At the very least I will emphasis this in the arg docuementation :D

Reply:
TBH, I was ready to trash all logic for `is_spm_file`.